uber/tests/models/test_job.py

```diff
@@ -46,7 +46,8 @@ def test_non_volunteer(self, session):
 
     def test_restricted(self, session):
         assert session.assign(session.staff_two.id, session.job_six.id)
-        session.staff_two.trusted = True
+        session.staff_two.trusted_depts = str(c.CONSOLE)
+        # TODO: update
         session.commit()
         assert not session.assign(session.staff_two.id, session.job_six.id)
 
```

Comment:
It's set to trusted where it's defined at
https://github.com/magfest/ubersystem/blob/61c0a5f4360bf637545d239e58d2bd3afabf5eac/uber/tests/conftest.py#L120

It's also already got a location of ``c.CONSOLE`` so you don't need to set the job location.